packages/web/projects/web-components/src/components/feedback/toast/toast.component.ts

```diff
@@ -17,5 +17,13 @@ export class FreudToastComponent {
 
   @Input() position: positions = 'top-right';
   @Input() key!: string;
+  @Input() url!: string;
 
+  openURL(event: any): void {
+    const isCloseBtn = event.srcElement.className.includes('close-icon');
+
+    if(!isCloseBtn && this.url){
```

Comment:
Sugestão _(code style)_:
```suggestion
if (!isCloseBtn && this.url) {
```

Reply:
Feito